vagrant/dev/bootstrap.sh

```diff
@@ -5,17 +5,17 @@
 set -e
 
 # Install useful yum repos
-yum install -y https://s3.amazonaws.com/archive.zfsonlinux.org/fedora/zfs-release$(rpm -E %dist).noarch.rpm
-curl https://copr.fedoraproject.org/coprs/tomprince/hybridlogic/repo/fedora-20-x86_64/tomprince-hybridlogic-fedora-20-x86_64.repo >/etc/yum.repos.d/hybridlogic.repo
+yum install -y http://archive.zfsonlinux.org/epel/zfs-release$(rpm -E %dist).noarch.rpm
+yum install -y https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
 
 # Install packages
 yum install -y \
 	@buildsys-build git \
-	kernel-headers kernel-devel rpmdevtools \
+	dkms kernel-headers kernel-devel rpmdevtools \
```

Comment:
I would be inclined to move the `dkms` install to before the reboot, so the guest additions get installed with `dkms`.